CDP4CommonView/Items/CitationLayoutGroup.xaml

```diff
@@ -14,7 +14,7 @@
                   mc:Ignorable="d"
                   VerticalAlignment="Stretch"
                   HorizontalAlignment="Stretch"
-                  Width="500"
+                  Width="525"
```

Comment:
What happens if you remove the `width` completely? does it not auto size to fit the dialog?